Assets/Scripts/Player.cs

```diff
@@ -20,6 +20,20 @@ public class Player : MonoBehaviour
     Vector3 moveDirection = Vector3.zero;
     string lastAnimation = "";
 
+    // Alive interface
+    [SerializeField]
+    private int hitPoints;
+
+    public int HitPoints {
+        get { return hitPoints; }
+        set { hitPoints = value; }
+    }
```

Comment:
In case you didn't know that, you're implementing the default behaviour, so you could just write:

```cs
    public int HitPoints {
        get;
        set;
    }
```

Reply:
I do know that but the Unity compilator is complaining in that case because the Property is not set to some bullshit so I went for the explicit version